lib/please_cli/please_cli/config.py

```diff
@@ -236,11 +252,15 @@
                 'heroku_app': 'releng-staging-mapper',
                 'heroku_dyno_type': 'web',
                 'url': 'https://mapper.staging.mozilla-releng.net',
+                # TODO: switch to SSL Endpoint
```

Comment:
@dividehex can we switch to SSL Endpoint for ``releng-staging-mapper`` heroku app?